src/Raven.Server/Documents/PeriodicBackup/PeriodicBackupRunner.cs

```diff
@@ -634,13 +635,37 @@ private PeriodicBackupStatus GetBackupStatusFromCluster(long taskId)
             using (_serverStore.ContextPool.AllocateOperationContext(out TransactionOperationContext context))
             using (context.OpenReadTransaction())
             {
-                var statusBlittable = _serverStore.Cluster.Read(context, PeriodicBackupStatus.GenerateItemName(_database.Name, taskId));
+                return GetBackupStatusFromCluster(context, taskId);
+            }
+        }
+
+        private PeriodicBackupStatus GetBackupStatusFromCluster(TransactionOperationContext context, long taskId)
+        {
+            var statusBlittable = _serverStore.Cluster.Read(context, PeriodicBackupStatus.GenerateItemName(_database.Name, taskId));
+
+            if (statusBlittable == null)
+                return null;
+
+            var periodicBackupStatusJson = JsonDeserializationClient.PeriodicBackupStatus(statusBlittable);
+            return periodicBackupStatusJson;
+        }
 
-                if (statusBlittable == null)
-                    return null;
+        private long GetMinLastEtag()
+        {
+            var min = long.MaxValue;
 
-                var periodicBackupStatusJson = JsonDeserializationClient.PeriodicBackupStatus(statusBlittable);
-                return periodicBackupStatusJson;
+            using (_serverStore.ContextPool.AllocateOperationContext(out TransactionOperationContext context))
+            using (context.OpenReadTransaction())
+            {
+                var record = _serverStore.Cluster.ReadRawDatabaseRecord(context, _database.Name);
```

Comment:
When we have a backup with `Full Backup/Snapshot` without an incremental, we don't need to keep the tombstones at all.

Reply:
I think we do need to keep the tombstones... consider we have an ETL or external replication task that didn't sent the tombstones yet.

Reply:
We don't backup tombstones during a full backup

Reply:
done